themes/default/content/blog/azure-container-solutions/index.md

```diff
@@ -0,0 +1,182 @@
+
+If you're at the stage of trying to run your application in Microsoft Azure, the choices can be overwhelming. The Azure Container [product page](https://azure.microsoft.com/en-gb/product-categories/containers/) lists 7 different products on their landing page, and for new users it can often be difficult to decide which of the myriad products is right for their use case. What can make it even more confusing is that often these container services can be interoperable, meaning you can use one container product from another!
+
+In this post, we're going to examine each of the main container services offered in Azure and then examine what they're good for and what they might not be so good for. Let's take a look!
+
+<!--more-->
+
+## Azure Container Instances
+
+[Azure Container Instances](https://azure.microsoft.com/en-us/services/container-instances/) (ACI) bills itself as an offering that lets you "easily run containers on Azure without managing servers."
+
+Launched in July 2017, it was arguably the first "serverless" container offering from the major cloud providers.
+
+### What's good about it?
+
+ACI is a Platform-as-a-Service (PaaS) type of product that is designed to remove the burden of launching and running containers.
+
+#### Simplicity
+
+Azure Container Instances has been designed from the ground up to be straightforward to use. It’s designed to get your container launched as quickly as possible, even including integrations with many of the tools you’re probably already familiar with if you’re using containers. One example is the integration with the [Docker CLI](https://docs.microsoft.com/en-gb/azure/container-instances/quickstart-docker-cli).
+
+In many cases, the only configuration options you need to pass to your ACI provisioning mechanism is the image you’re running and some sizing decisions, like how many CPUs and how much memory you need. There is some flexibility in this decision process, but having fewer required options means you’re not bogged down with complex architectural decisions when trying to get your container running.
+
+This lends itself particularly well to web server projects, especially at smaller scales. You can launch applications very quickly and easily by specifying the image you've built, and your application can be running in very short order.
+
+#### Start-Up Times
+
+ACI Containers start fast&mdash;often in a matter of seconds. This speed really helps the developer lifecycle and helps to create productivity when you're developing your application.
+
+#### Pricing
+
+ACI will only bill you for what you use. It offers no start-up costs (which goes great with the fast start times!) and per-second billing, meaning you only pay for what you use. The pricing is also extremely competitive against other cloud providers.
```

Comment:
I find "pay for what you use" a bit confusing. You pay per time regardless of whether there are any users/requests to the container, as opposed to more serverless services that charge per request and can scale to zero.